sass/px-data-grid-dark-theme.scss

```diff
@@ -42,4 +42,5 @@ html {
   --px-data-grid-default-highlight-background-color       :  $gray3;
   --px-data-grid-yellow-status-background-color           :  $status-yellow4;
   --px-data-grid-red-status-background-color              :  $status-red4;
+  --px-auto-filter-action-bar-bottom-spacing              :  20px;
```

Comment:
dark-theme and light-theme are only for colors. Do we really need this variable?

Reply:
Maybe not, I will remove it.